samcli/local/common/runtime_template.py

```diff
@@ -164,5 +164,5 @@ def get_local_lambda_images_location(mapping, runtime):
     "python3.6": "Python36",
     "python3.8": "Python36",
     "python3.9": "Python36",
-    "dotnet6": "dotnetcore3.1",
```

Comment:
Dotnet is currently not a supported language for EventBridge Schemas code bindings

Reply:
minor - If it's not supported, shall we remove this line? Even if it will be supported in the future, I bet the language code would be "dotnetcore3.1". It doesn't hurt to still keep it here as it doesn't impact anything, but just create confusion.

Reply:
I think some other tests rely on this value being here. I tried removing the value and some of the checks failed.